volumes/vds/readme.md

```diff
@@ -0,0 +1,448 @@
+| cross_road       | text              | BATHURST STREET       |               |
+| district         | text              | Toronto and East York |               |
+| ward             | text              | Trinity-Spadina (20)  |               |
+| vds_type         | text              | inductive_loop        | Other values: microwave_radar (origin of classification unknown) |
+| total_loops      | smallint          | 6                     |               |
+| sequence_number  | text              | LAKE173               |               |
+| data_range_low   | integer           | 20000                 |               |
+| data_range_high  | integer           | 30000                 |               |
+| historical_count | integer           | 9700                  |               |
+| arterycode       | integer           | 826                   |               |
+
+# Tips for Use
+- Start with the timeframe of interest and identify which sensors have data available rather than first identifying the correct sensors. 
+- The raw data tables are very large. Make sure to use Explain (F7) and check you are using available indices and partitions.
+- In some cases you may find it easier to select from only the partition of interest. eg. `FROM vds.raw_vdsdata_div2_202308` instead of more verbose ```FROM vds.raw_vdsdata WHERE division_id = 2 and dt >= '2023-08-01 00:00:00'::timestamp....```.
+- For RESCU requests, make use of the manually defined fields in `vds.detector_inventory` for easy filtering.
+
+# DAG Design 
+VDS data is pulled daily at 4AM from ITS Central database by the Airflow DAGs described below. The dags need to be run on-prem to access ITSC database and are hosted on Morbius. 
+
+## [vds_pull_vdsdata](../../dags/vds_pull_vdsdata.py)
+<div style="width: 75%";>
+
+  ![Alt text](vds_pull_vdsdata_dag.png)
+
+</div>
+
+- `check_partitions` checks the necessary partitions are available for `raw_vdsdata`, `counts_15min`, `counts_15min_bylane` before pulling data. 
+
+**pull_vdsdata**  
+- `delete_vdsdata` deletes existing data from RDS `vds.raw_vdsdata` to enable easy rerunning of tasks.  
+- `pull_raw_vdsdata` pulls into RDS from ITS Central database table `vdsdata` including expanding binary `lanedata` column. After insert, a trigger function adds foreign keys referencing `vdsconfig` and `entity_locations` tables. 
+
+**summarize_v15**  
+- `summarize_v15` first deletes and then inserts a summary of `vds.raw_vdsdata` into `vds.counts_15min`.  
+- `summarize_v15_bylane` first deletes and then inserts summary of `vds.raw_vdsdata` into `vds.counts_15min_bylane`. 
+
+**update_inventories**  
+These tasks are arbitrarily grouped under this DAG because they run on the same schedule (daily) and with the same connections. They could instead be in a separate DAG if desired. 
+- `skip_update_inventories` task ensures these only run for the most recent schedule interval (doesn't backfill). 
```

Comment:
ooh you gotta show us more of this in the presentation, probably other applications out there.